### ISSUE TITLE
Slow loading of Tar page

### ISSUE THREAD

**Issue by JasonLing95** [eastgenomics/Ploutos#49]
![Screenshot from 2023-03-03 11-55-46](https://user-images.githubusercontent.com/60819172/222715063-2b73ce17-572d-4c6d-b5c7-e05979fb2db4.png)

**Comment by jethror1** [comment]
are you limiting the loading of records with datatables? If not it will try load all the data in one go

**Comment by JasonLing95** [comment]
yes there's pagination at the frontend. problem lies in backend (Django) sending a full HTML to the frontend to render causing it to sometimes be a bit slow to render (this is for project-level-detail)

Specifically for tar page, the problem lies in DNAnexus API query being slow. we can speed this up using cache
